sklearn/svm/classes.py

```diff
@@ -578,6 +594,11 @@ class SVC(BaseSVC):
         implemented using liblinear. Check the See also section of
         LinearSVC for more comparison element.
 
+    References
```

Comment:
Shoud Platt (2000) be added as well? i.e. *"Probabilistic outputs for support vector machines and comparison to regularizedlikelihood methods."*